pkg/provider/onepassword/onepassword_test.go

```diff
@@ -0,0 +1,362 @@
+/*
```

Comment:
Tests look good.  Only minor (and optional) nit I have is they might be easier to read if they were re-written with a table-driven style.

Reply:
re-written

Reply:
Finished the rest. Everything is table driven now.